tasks/generate_cert_fixtures.rake

```diff
@@ -0,0 +1,43 @@
+desc "generate cert fixtures"
+task(:gen_cert_fixtures) do
```

Comment:
What's the use case for this? Is this just in case someone wants to update the test fixtures? It'd be nice if you could add a little more detail about why you added this new task and under what scenarios someone might want to run it

Reply:
I added some comments to this file.